### PR TITLE
Upgrade version of dev dependencies

### PR DESCRIPTION
**Pull request by samnela**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -8,7 +8,6 @@
     convertWarningsToExceptions="true"
     processIsolation="false"
     stopOnFailure="false"
-    syntaxCheck="false"
```
Comment (samnela) [inline phpunit.xml.dist:11]: The attribute 'syntaxCheck' is not allowed on phpunit ^7

File: composer.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
         "swarrot/swarrot": "~2.0|~3.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~1.12",
-        "phpunit/phpunit": "^5.7"
+        "friendsofphp/php-cs-fixer": "^1.12|^2.0",
+        "phpunit/phpunit": "^5.7|^6.0|^7.0"
```
Comment (odolbeau) [inline composer.json]: I think you can force the latest version `^7.0` only. (Same for php-cs-fixer)

File: composer.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
         "swarrot/swarrot": "~2.0|~3.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~1.12",
-        "phpunit/phpunit": "^5.7"
+        "friendsofphp/php-cs-fixer": "^1.12|^2.0",
```
Comment (odolbeau) [inline composer.json]:
```suggestion
        "friendsofphp/php-cs-fixer": "^2.0",
```

File: composer.json (hunk 2 above)
Comment (odolbeau) [inline composer.json:14]:
```suggestion
        "phpunit/phpunit": "^7.0"
```

Comment (samnela) [reply composer.json:14]: Should I drop support for php 5.6 to 7.0 ? (see  .travis.yml )

Comment (odolbeau) [reply composer.json:14]: We'll do this in another PR, forget my suggested changes!

We'll need:
* to remove support of PHP 5.6 / 7.0
* to bump all requirements (including non-dev)

if we add strict casting: to release a new major version.

I merge your PR right now, feel free to open another one if you want to remove php 5.6 / 7.0 support.

**Pull request merged**